### PR TITLE
fix: Update Helm ingress template & document

### PR DESCRIPTION
**Pull request by geekup-legodevops**
## Description
- Change path type in rule of ingress template to `Prefix`
- Update document for Helm installation without domain
## Type of change
- Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: deploy/helm/README.md [hunk 1]
```diff
@@ -217,7 +217,10 @@ helm install stable-appsmith/appsmith --generate-name \
 ```
 - If you have installed Appsmith Helm chart, please run the `helm upgrade` command to upgrade the existing installation
 ```
-helm upgrade --set ingress.enabled=true stable-appsmith/appsmith appsmith
+helm upgrade stable-appsmith/appsmith appsmith \
+--set ingress.enabled=true \
+--set ingress.annotations."kubernetes\.io/ingress\.class"=nginx \
+--set service.type=ClusterIP
```
Comment (sharat87) [inline deploy/helm/README.md:223]:
```suggestion
  --set ingress.enabled=true \
  --set ingress.annotations."kubernetes\.io/ingress\.class"=nginx \
  --set service.type=ClusterIP
```

File: deploy/helm/templates/ingress.yaml [hunk 2]
```diff
@@ -50,7 +50,7 @@ spec:
       http:
         paths:
           - path: /
-            pathType: ImplementationSpecific
+            pathType: Prefix
```
Comment (sharat87) [inline deploy/helm/templates/ingress.yaml:53]: Can you elaborate a little on what this change does please?

Comment (geekup-legodevops) [reply deploy/helm/templates/ingress.yaml:53]: Hi @sharat87 Reference to the document of K8S about this (https://kubernetes.io/docs/concepts/services-networking/ingress/#path-types). Actually, `ImplementationSpecific` will depends on the IngressClass to treat path as `Prefix` or `Exact`. In our application, we will have a variety of request path -> Therefore, we prefer `Prefix` path type to ensure our application can treat request path properly

**Comment by sharat87** [comment]
@geekup-legodevops, do we need to increment the version of Helm charts in this PR?

**Comment by geekup-legodevops** [comment]
> @geekup-legodevops, do we need to increment the version of Helm charts in this PR?
Yes @sharat87 we will increase it to version 1.5.2

**Review by sharat87 — APPROVED** [review]

**Pull request merged**